package.json

```diff
@@ -2,11 +2,13 @@
   "name": "optic-release-automation-app",
   "version": "0.4.26",
   "description": "",
-  "main": "index.js",
+  "exports": "./index.js",
+  "type": "module",
+  "node": ">=20",
```

Comment:
@simoneb is restricting this to node 20+ a problem? I went off the work we were doing in Fastify.

Reply:
I don't think so, but let's make sure we're deploying to a recent version of Node in cloud run. See the cd workflow, we don't seem to be specifying a version there